src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/DataSetViewModels.cs

```diff
@@ -36,3 +38,27 @@ public record DataSetLiveVersionViewModel : DataSetVersionViewModel
 {
     public required DateTimeOffset Published { get; init; }
 }
+
+public record DataSetVersionSummaryViewModel
+{
+    public required string Title { get; init; }
+
+    public required Release Release { get; init; }
```

Comment:
Nick suggested returning the `Release.Id` in case we want to link directly to a release from here in the future.

In terms of naming, I reckon `ReleaseViewModel` is a better shout! I'm not a fan of `IdTitleViewModel` as that name could change if the properties changed. But I see where you're coming from!

Reply:
After discussions yesterday, we're no longer returning the `Release` details. So won't need a `ReleaseViewModel`. We will now be returning `ReleaseVersion` details instead. So I will add a `ReleaseVersionViewModel`